lib/fs.js

```diff
@@ -1125,32 +1124,10 @@ function rmdir(path, options, callback) {
   callback = makeCallback(callback);
   path = getValidatedPath(path);
 
-  if (options?.recursive) {
```

Comment:
I would actually prefer if this throwed an exception now - silently ignoring is probably not easy to spot.

Reply:
Likely yes but I think it's reasonable to throw a more specific error here.